src/containers/Form.js

```diff
@@ -138,6 +138,8 @@ Form.propTypes = {
   submitButton: PropTypes.object,
   initialValues: PropTypes.object,
   validationMeta: PropTypes.object,
+  otherNetworkEntities: PropTypes.object,
+  subject: PropTypes.object.isRequired, // Implicit dependency used by autoInitialisedForm
```

Comment:
I'm getting a console warning related to `subject`:

```
Warning: Failed prop type: The prop `subject` is marked as required in `Form`, but its value is `undefined`.
```

Reply:
fixed with [3da647e](https://github.com/complexdatacollective/Interviewer/pull/1242/commits/3da647ec3fc0f84b08e77f8fe9801c4a5e64d288)